src/client/src/translations/en.json

```diff
@@ -104,6 +104,7 @@
   "pages.limitedPagesMessage": "You can select up to 20 pages",
   "pages.maxPagesText": "Icon for Max Pages Description",
   "pages.overlimitPagesMessage": "You cannot add more than 20 pages to the project",
+  "pages.noPageGeneration": "You cannot add more than 20 pages to the project",
```

Comment:
This doesn't look right.. should the message be the same as https://github.com/microsoft/WebTemplateStudio/pull/913/files#diff-abf0943cba1cc720f993c71a080c2f04R32 ?

Reply:
Yeah, I will update it.